actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsActor.java

```diff
@@ -91,7 +94,9 @@ private void orgConsumptionMetricsReport(Request actorMessage) {
   }
 
   private String createReportTrackingEntry(Request actorMessage) {
-    ProjectLogger.log("Create Report Tracking Entry");
+    ProjectLogger.log(
+        "OrganisationMetricsActor:createReportTrackingEntry report tracking method call start.",
```

Comment:
From:
"OrganisationMetricsActor:createReportTrackingEntry report tracking method call start."

To:
"OrganisationMetricsActor: createReportTrackingEntry called."